HW_1/task_1.py

```diff
@@ -0,0 +1,14 @@
+#  Задача 2: Найдите сумму цифр трехзначного числа.
+
+#  123 -> 6 (1 + 2 + 3)
+#  100 -> 1 (1 + 0 + 0)
+
+
+num = int(input('Ведите число:'))
+digitsum = 0
+
+while num > 0:
+    digitsum = digitsum + num % 10
```

Comment:
Можно покороче записать
```suggestion
    digitsum += num % 10
```